static/styles/settings.css

```diff
@@ -925,6 +925,28 @@ input[type="checkbox"] {
     }
 }
 
+.field.password-div {
+    position: relative;
+
+    .password_visibility_toggle {
+        position: absolute;
+        left: 190px;
+        top: 30px;
+    }
+}
+
+#api_key_form {
+    .password-div {
+        position: relative;
+
+        .password_visibility_toggle {
+            position: absolute;
+            left: 200px;
+            top: 30px;
+        }
+    }
+}
```

Comment:
This is almost a duplicate of the previous block -- can it be deleted?  The only difference is 10px in the `left` attribute.

Reply:
In an ideal world we'd just have a single copy of this in the shared `components.css`, if we can make the same CSS work for all the different fields.